util/util.go

```diff
@@ -33,10 +33,8 @@ type KernelDevice struct {
 
 func getIPFromAddrs(addrs []net.Addr) string {
 	for _, addr := range addrs {
-		if ip, ok := addr.(*net.IPNet); ok && !ip.IP.IsLoopback() {
-			if ip.IP.To4() != nil {
-				return strings.Split(ip.IP.String(), "/")[0]
-			}
+		if ip, ok := addr.(*net.IPNet); ok && !ip.IP.IsGlobalUnicast() {
```

Comment:
? Should be `ip.IP.IsGlobalUnicase()`?

Reply:
Hi,
According to the doc (https://golang.org/pkg/net/#IP.IsGlobalUnicast) it's 'IsGlobalUnicast'

Reply:
Well, that was embarrassing..
Thanks for the review !

Reply:
Ah, made a typo too :)